src/libpfc.c

```diff
@@ -382,7 +382,28 @@ static const struct EVENT EVENT_LIST[256] = {
     {0x00, NULL             , NULL}
 };
 
+/* Error Codes */
+#define PFC_ERR_OPENING_SYSFILE (-10)
```

Comment:
Here are the error codes, but I should probably move them to the libpfc.h header file so clients could check them directly?